tests/support/helpers.py

```diff
@@ -747,6 +748,17 @@ def wrap(cls):
                                 username
                             )
                         )
+            if not salt.utils.platform.is_windows() and password is not None:
+                if salt.utils.platform.is_darwin():
+                    hashed_password = password
+                else:
+                    hashed_password = salt.utils.pycrypto.gen_hash(
+                        crypt_salt="SALTsalt", password=password
```

Comment:
@DmitryKuzmenko We should probably not use a static salt here. It'd be better to have this as a random string in a config file which a user can update. That way they'll actually get some security from using a salted hash ;)

Reply:
=)))

Reply:
Always be as safe as you can ))

Reply:
Actyally we can not set the salt here, but this will force `crypt` to generate more secure salt that will make tests a little bit longer. So my way is more productive for tests. :-P